dsl/procedures/Deploy/steps/Deploy.pl

```diff
@@ -0,0 +1,2 @@
+$[daticalInstallPath]\hammer.bat --project $[daticalProjectPath] --drivers $[daticalPluginsPath] deploy $[daticalDeploymentStep]
+
```

Comment:
Do you mean issue #4? the one for Linux support? Issue #1 was about the link to the report and it was fixed